app/src/main/res/values/strings.xml

```diff
@@ -138,4 +138,15 @@
   <string name="chip_option">Option</string>
   <string name="chip_select">Select</string>
   <string name="chip_dismiss">Dismiss</string>
+  <string name="default_text">Default</string>
```

Comment:
maybe prefix these with `input`?

Reply:
We already have "Default" in strings.xml – in fact, in 2 places. Let's make it generic and reusable.

Reply:
duplication in strings files isn't necessarily a bad thing as they might be used in different cases. if something is generic i'd mark it as generic and prefix it that way (and remove the duplicates at the same time), rather than implicitly making all of the input strings generic

Reply:
In this case, we're working with the demo app. Using XML strings create some complexity itself. Considering that we don't get a lot of value from this (we're using this as an example of proper locale handling) I'd rather not increase the complexity further.

Reply:
let's keep it simple and rename them then. just because it's a demo app doesn't mean we shouldn't follow good practices

Reply:
We are following good practices. I'm just saying we need to reuse the strings when possible to not increase the complexity of the strings management. We do this for a lot of other components, the only reason why we can't do this is that 'default' is a reserved Java keyword – hence the suffix.

Reply:
all our strings are prefixed with something right now - again: if they should be generic, let's also indicate that they are and update other usages. otherwise we're just adding another version of the same string with no benefit

Reply:
We can a generic prefix, yes. Which would you prefer?